lib/repository/interface/movie_interface.dart

```diff
@@ -0,0 +1,6 @@
+import '../../model/movie_data.dart';
+
+abstract class MovieInterface {
+  Future<Movie> getMovieById(id);
```

Comment:
Please specify the type of the "id" argument

Reply:
Done